app/controllers/addresses_controller.rb

```diff
@@ -0,0 +1,23 @@
+class AddressesController < ApplicationController
+  def create
+    @user = Current.user
```

Comment:
Are you using this `@user` value somewhere?(except the next line). If not, do we really need to initialize this?

Reply:
No. variable removed